### ISSUE TITLE
lr_scheduler.OneCycleLR

### ISSUE THREAD

**Issue by skeydan** [mlverse/torch#248]
could we have that scheduler please? (using it in the book)
